pom.xml

```diff
@@ -248,3 +248,4 @@
         </plugins>
     </build>
 </project>
+
```

Comment:
What it the benefit of the additional new line at end of file?

Reply:
the purpose of this pr was to trigger the IT, not to propose a change.